go-controller/pkg/metrics/cluster_manager.go

```diff
@@ -54,6 +54,30 @@ var metricV6AllocatedHostSubnetCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The total number of v6 host subnets currently allocated",
 })
 
+/** EgressIP metrics recorded from cluster-manager begins**/
```

Comment:
Add the updates your doing to the metrics doc in /docs/. Downstream folks rely on that doc to adjust their testing  after metrics change.

Reply:
ack, let me do that ...